### DISCUSSION TITLE
Translating V to C#/MSIL

### DISCUSSION THREAD

**Discussion by wond12** [vlang/v#11585] [Ideas]
Is it possible to translate V to C# or MSIL, so that v could be used in unity.

**Comment by kinggoesgaming** [comment]
This honestly will need LLVM-isque infrastructure for it to be possible, as doing so we are entering the world of building multiple targets

**Comment by streaksu** [comment]
@kinggoesgaming LLVM compiles to native object code for several platforms, but MSIL is not native neither object code, but a bytecode used specifically for the .NET environment.
Such a task would require a specific backend to MSIL, which right now is not feasible in the state of the compiler imho, there is just too much to fix for this to be a real proposition.

**Comment by medvednikov** [comment]
This is definitely doable.
